README.md

```diff
@@ -59,6 +59,13 @@ Every argument is optional.
 | `exempt-all-pr-assignees`         | Exempt all PRs with assignees from being marked as stale.<br>_Override `exempt-all-assignees`_.                                                               |
 | `enable-statistics`               | Display some statistics at the end of the logs regarding the stale workflow.<br>Only when the logs are enabled.<br>_Defaults to **true**_.                    |
 
+### List of output options
+
+| Output        | Description                |
+| ------------- | -------------------------- |
+| staled-issues | List of all staled issues. |
```

Comment:
should we call these something more generic to handle the fact that these can be either an `issue` or a `pull request`? I know it's confusing since PR's "are" issues. But maybe just better to call it out so someone isn't surprised to see PRs in the output.

Reply:
So I will add `prs` to the naming so it must be clear.